examples/browser/webpack.config.js

```diff
@@ -13,7 +13,7 @@ const CircularDependencyPlugin = require('circular-dependency-plugin');
 
 const outputPath = path.resolve(__dirname, 'lib');
 
-const monacoEditorPath = '../../node_modules/monaco-editor-core/min/vs';
+const monacoEditorPath = '../../node_modules/monaco-editor-core/dev/vs';
```

Comment:
Do we want to switch back to `dev`?

Reply:
I think for us we can stick to `dev`, in the generator it is still `min`, or it can depend on NODE_ENV variable. Do we have an issue for it?

Reply:
Not at all, I did the same change locally, that's why I have asked.

Reply:
By the way, I do the same for the electron one.